Daf.Meta.Editor/UserControls/HubRelationshipsControl.xaml.cs

```diff
@@ -11,5 +12,20 @@ public HubRelationshipControl()
 		{
 			InitializeComponent();
 		}
+
+		private void ComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+		{
+			// This event is called both when a StagingColumn is added/removed from StagingControl AND
+			// when the user changes the selection in the HubRelationshipsControl. It causes binding errors
+			// when StagingColumns are removed, but works as intended when the user manually changes
+			// the ComboBox.SelectedItem.
```

Comment:
It's a bit of a crude solution, but it appears to work correctly as it is. Ideally I would only want it to be called when the user changes the selection. However it might be "good enough" for now and something that I can fix when I look into fixing the issue of being able to assign the same Staging Column to more than one Hub. Any solution implemented now is likely to be affected then.